src/pages/hegel/reference/nothing/difference-between-being-and-nothing.mdx

```diff
@@ -0,0 +1,202 @@
+    Houlgate's argument both for simplification and because it is contentious
+    (Niklas).
+
+Houlgate uses the term "preserve" alongside "restore" in his exposition of this
+argument. However, the term "preserve" suggests that the _singularly_ same
+immediate difference between `being` and `nothing` is carried through its
+disappearance. But there is no distinguishing factor either for or against this;
+the difference is exactly immediate and so nothing determinate can be made out
+about it that would definitively show that it is one or the other, namely, that
+it is _this_ immediate difference between `being` and `nothing` or _the same
+type of_ immediate difference between `being` and `nothing` (as being the
+instance of a class). For all intends and purposes, this matter may be
+inconsequential but it is not unambiguous.
+
+{/* TODO Revisit this when commentary on the Concept is written. Trisokkas uses the term "preserve" as a peculiar characteristic of the concept. */}
+
+### The Dynamism of Being
+
+> If the result that being and nothing are the same seems inherently startling
+> or paradoxical, there is not much to be done about it. We should be amazed
+> rather at this amazement that appears so refreshing in philosophy... (Hegel
+> 2010, 61/21.71).
+
+The opening of the _Science of Logic_ quickly violates the principle of
+non-contradiction or the law of identity. The form of thinking involved in
+Hegel's logic disrupts that kind which clings on to clear and definite
+distinctions between `being` and `nothing`, such as that found in Parmenides
+which resists the thought that they could ever be the same (Houlgate 2022, 146).
+The idea that a thing could only ever be the thing that it is, or whose being is
+identical to itself, is disproven at the very outset of the _Logic_, for at the
+very least `being` (and `nothing`) is unable to _simply be_ &ndash; it vanishes
+into `nothing`.
+
+As Houlgate describes it, there is nothing mysterious or irrational about the
+dialectical change taking place in `being` and `nothing` into each other. "That
+dialectical conversion is logically necessary: being and nothing pass into one
+another for the reasons we have seen, and the immediate difference between them
+thereby proves to be, of necessity, an indeterminate, unstable difference"
+(Houlgate 2022, 146). Hegel therefore does not mean to flout the traditional
+principles of reasoning or to undermine rational argument, he is simply
+revealing the "dynamism in being that he takes to become evident when one
+focuses on being in its purity and immediacy _without uncritically assumed
+preconceptions_" (Houlgate 2022, 146). When `being` is considered
+[without presuppositions](/hegel/guides/presuppositionless-thinking), the
+argument of the _Logic_ is unassailable: `pure being` is utterly incapable to
+simply be.
+
+### Parallel Categories, Divergent Conceptions (Niklas)
+
+When regarding the respective transitions of `being` and `nothing`, note that in
```

Comment:
there's a syntactical error in this sentence. Some kind of connector or preposition is missing.

Reply:
Thanks for spotting that one! 🌞